frontend/src/components/Profile/Profile.tsx

```diff
@@ -76,7 +97,7 @@ export const ProfileView = (data) => {
                                         {score.block_name}
```

Comment:
This line seems to indicate that the rename to `block_name` has already been applied?

Reply:
Yeah, that's because this branch has been rebased already on the latest develop changes